shell/browser/native_window_views.cc

```diff
@@ -757,6 +759,20 @@ void NativeWindowViews::SetAlwaysOnTop(bool top,
     NativeWindow::NotifyWindowAlwaysOnTopChanged();
 
   widget()->SetAlwaysOnTop(top);
+
+#if defined(OS_WIN)
+  // Reset the placement flag.
+  behind_task_bar_ = false;
+  if (top) {
+    // On macOS the window is placed behind the Dock for the following levels.
+    // Re-use the same names on Windows to make it easier for the user.
+    const std::vector<std::string> levels = {
```

Comment:
don't allocate this on the heap unnecessarily
```
static const auto levels[] = {
        "floating", "torn-off-menu", "modal-panel", "main-menu", "status"};
```

Reply:
Done